.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -13,4 +13,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/wrapper-validation-action@v1.0.5
```

Comment:
v1 is a reference style for major version. I remember we can control whether using latest patch version if possible.